zerver/models.py

```diff
@@ -1428,12 +1428,13 @@ def content_has_link(content: str) -> bool:
                 'bitcoin:' in content)
 
     @staticmethod
-    def is_status_message(content: str, rendered_content: str) -> bool:
+    def is_status_message(content: str, rendered_content: Optional[str]) -> bool:
```

Comment:
I think this function should expect to be passed (str, str); I bet the caller should be doing an `is not None` assert.